store_monitoring/crud.py

```diff
@@ -41,4 +51,40 @@ def insert_bq_results(db):
             db.commit()
             db.refresh(data)
     return
-        
\ No newline at end of file
+
+def current_time():
```

Comment:
Again, by the name i thought it's gonna give me current time but it's giving me something else in return.

So rename the function